boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20-memory_map.dtsi

```diff
@@ -51,6 +51,23 @@
 			};
 		};
 
+		etr_buf_ram0x_region: memory@2f0be000 {
```

Comment:
If we move it to the start of `cpuapp_ram0x_region` it would create less friction when resizing app and radio core partitions. It can just be 'in' the region as well so we safe a mpc override
https://github.com/zephyrproject-rtos/zephyr/pull/72476/files#diff-9b8c6027f61ec3ca7e9e8de6dc24eeb62fe40bd165cd832d845de701c3b496a0R21